packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -3363,7 +3363,7 @@ describe('network stubbing', { retries: 15 }, function () {
           done()
         })
 
-        cy.intercept('/timeout*', (req) => {
+        cy.intercept('/timeout*', { times: 1 }, (req) => {
```

Comment:
This change was to fix the flake. I would like to understand this problem better and fix the root cause hopefully in a follow up PR.

Reply:
can you add a TODO or NOTE to this test indicating the change and the expected followup?

Reply:
Added: [`d0415ce` (#27140)](https://github.com/cypress-io/cypress/pull/27140/commits/d0415ced407df451fcea7502c0d794967ae0cbe3)